node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -172,6 +187,31 @@ impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
         Self::remove_expired(&mut self.vertex_deps, oldest);
     }
 
+    fn vertices_to_be_added_later_size(&self) -> u64 {
```

Comment:
I'd use `len` instead of `size` here, to avoid confusion with the `DataSize` outputs.

Reply:
Renamed in [`878c031` (#1159)](https://github.com/CasperLabs/casper-node/pull/1159/commits/878c03148fec1cbc6cae2072eafc6271dd88bd36)